packages/fxa-settings/src/lib/file-utils-legal.tsx

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { determineLocale } from 'fxa-shared/l10n/determineLocale';
+import sentryMetrics from 'fxa-shared/lib/sentry';
+
+export enum LegalDocFile {
+  privacy = 'firefox_privacy_notice',
+  terms = 'firefox_cloud_services_tos',
+}
+
+const LEGAL_DOCS_PATH = '/settings/legal-docs';
+
+// TODO: probably move this + clone script to gql-api to reduce network requests
```

Comment:
I did it this way originally because it was easy to mostly copy the l10n clone script for our legal-docs script and I saw we were using `fetch` elsewhere. After I had it mostly working I realized it should probably go in gql-api instead especially this makes two network requests. I'll file an issue for cleanup here.

Reply:
Follow up: https://mozilla-hub.atlassian.net/browse/FXA-6809